RecoLocalCalo/EcalRecAlgos/interface/EcalUncalibRecHitRecAnalFitAlgo.h

```diff
@@ -119,13 +128,13 @@ class EcalUncalibRecHitRecAnalFitAlgo : public EcalUncalibRecHitRecAbsAlgo<C> {
     pedestal.SetParameter(0, frame[0]);
     pedestal.SetParName(0, "Pedestal");
 
-    graph.Fit(&pulseShape, "QRM");
+    int result = graph.Fit(&pulseShape, "QRMN SERIAL");
     //TF1 *pulseShape2=graph.GetFunction("pulseShape");
```

Comment:
... and maybe take the occasion to clean up this one as well

Reply:
done